app/scripts/services/catalog.js

```diff
@@ -2,10 +2,19 @@
 
 angular.module("openshiftConsole")
   .factory("CatalogService", function($filter,
```

Comment:
Should we be concerned that there is already a "CatalogService" in origin-web-catalog?  https://github.com/openshift/origin-web-catalog/blob/master/src/services/catalog.service.ts#L4